### DISCUSSION TITLE
HTML code in the copyright

### DISCUSSION THREAD

**Discussion by cmaureir** [pradyunsg/furo#412] [Potential Issue]
Hello :wave:,

First of all thanks for this theme, it has been a really pleasant experience!
the looks and functionality are way superior to the one we had :100: 

I'm one of the [Qt for Python](https://pyside.org) developers, and for the new version that will be released in the following weeks, we decided to migrate our self-made theme to furo.

In the copyright string, we have the following substring:
```
The documentation provided herein is licensed under the terms of the <a href="https://www.gnu.org/licenses/fdl.html">GNU Free Documentation License version 1.3</a> as published by the Free Software Foundation.
```
which is mainly to link to the license, but I noticed that all the strings, including the `copyright` one, are being escaped:
https://github.com/pradyunsg/furo/blob/main/src/furo/theme/furo/page.html#L140

Using
```
{% trans path=pathto('copyright'), copyright=copyright|safe -%}
```
instead of
```
{% trans path=pathto('copyright'), copyright=copyright|e -%}
```
solves the issue, but that will completely break the norm on how the other strings are being handle.

So my question is, do you think this should not be allowed?, or maybe could be considered in future versions?

I thought about an additional option to avoid escaping, but that might motivate people to have every-other-string to not be escaped as well.

You can check the old footer [here](https://doc-snapshots.qt.io/qtforpython-6.2/index.html), and the snapshot of the new one [here](https://doc-snapshots.qt.io/qtforpython-dev/index.html).

Thanks for your time :)

**Comment by pradyunsg** [comment]
> So my question is, do you think this should not be allowed?

Maybe? I'm just copying what every other Sphinx theme does, so ideas for overrides are welcome. :)

> Maybe could be considered in future versions?

Yes. I'm open to ideas for this.
